driver.go

```diff
@@ -302,7 +330,10 @@ func (d *Driver) PreCreateCheck() error {
 	}
 
 	// GetPackage (and CreateMachine) both support package names and UUIDs interchangeably
```

Comment:
This comment isn't applicable anymore, right? (since we're now using `compute.GetPackageInput`, which appears to be explicit about that fact that we're passing it an ID instead of a name?)

Reply:
I think there might need to be a commit made that removes unnecessary code comments. I don't believe I got around to the others.